src/super_gradients/modules/quantization/quantized_stdc_blocks.py

```diff
@@ -0,0 +1,116 @@
+import torch
+from torch import nn
+
+from super_gradients.modules import Residual
+from super_gradients.training.models.segmentation_models.stdc import (
+    STDCBlock,
+    AttentionRefinementModule,
+    FeatureFusionModule,
+    ContextPath,
+)
+
+try:
+    from super_gradients.training.utils.quantization.core import SGQuantMixin, QuantizedMetadata
+    from super_gradients.training.utils.quantization.selective_quantization_utils import register_quantized_module
+
+    _imported_pytorch_quantization_failure = None
+except (ImportError, NameError, ModuleNotFoundError) as import_err:
+    _imported_pytorch_quantization_failure = import_err
+
+
+@register_quantized_module(float_source=STDCBlock, action=QuantizedMetadata.ReplacementAction.REPLACE_AND_RECURE)
+class QuantSTDCBlock(SGQuantMixin, STDCBlock):
+    if _imported_pytorch_quantization_failure is not None:
+        raise _imported_pytorch_quantization_failure
+
+    def __init__(self, in_channels: int, out_channels: int, steps: int, stdc_downsample_mode: str, stride: int):
+        super(QuantSTDCBlock, self).__init__(
+            in_channels=in_channels, out_channels=out_channels, steps=steps, stdc_downsample_mode=stdc_downsample_mode, stride=stride
+        )
+        self.quant_list = nn.ModuleList()
+
+        for i in range(len(self.conv_list) - 1):
+            self.quant_list.append(Residual())
+
+    def forward(self, x):
+        out_list = []
+        # run first conv
+        x = self.conv_list[0](x)
+        out_list.append(self.skip_step1(x))
+
+        for conv, quant in zip(self.conv_list[1:], self.quant_list):
+            x = conv(x)
+            out_list.append(quant(x))
+
+        out = torch.cat(out_list, dim=1)
+        return out
+
+
+@register_quantized_module(float_source=AttentionRefinementModule, action=QuantizedMetadata.ReplacementAction.REPLACE_AND_RECURE)
+class QuantAttentionRefinementModule(SGQuantMixin, AttentionRefinementModule):
```

Comment:
Some types has prefix Quantized (QuantizedMetadata) and some - Quant (Like current class).
I think for sake of consistency it would be great to name them all using same convention to avoid confusion why some starts with Quantized and some with Quant.

Reply:
QuantizedMetadata is a separate entity which is a part of the quantizer infra
Quant* are custom quantizeable counterparts for regular module